common/lib/xmodule/xmodule/course_module.py

```diff
@@ -437,152 +439,308 @@ class CourseFields(object):
                 ],
             },
             {
-                "short_description": _("Draft a Rough Course Outline"),
+                "short_description": _("Course Syllabus"),
                 "items": [
                     {
-                        "short_description": _("Create Your First Section and Subsection"),
-                        "long_description": _("Use your course outline to build your first Section and Subsection."),
+                        "short_description": _("Include Syllabus in Course Handouts"),
+                        "long_description": _("Include a separate static page or PDF for the Course syllabus."),
```

Comment:
Upload a PDF and then create a static page for it or add it as a course handout.